bulk/platform_vocab.csv

```diff
@@ -71,5 +71,17 @@ Global Station Papa Apex Profiler Mooring,WHOI,GP02HYPM
 Global Station Papa Flanking Subsurface Mooring A,WHOI,GP03FLMA
 Global Station Papa Flanking Subsurface Mooring B,WHOI,GP03FLMB
 Global Station Papa Gliders,Teledyne Webb,GP05MOAS
-Coastal Pioneer Central Profiler Mooring,WHOI,CP01CNPM
-Coastal Pioneer Inshore Profiler Mooring,WHOI,CP03ISPM
\ No newline at end of file
+Coastal Pioneer NES Central Profiler Mooring,WHOI,CP01CNPM
+Coastal Pioneer NES Inshore Profiler Mooring,WHOI,CP03ISPM
+Coastal Pioneer MAB Central Surface Mooring,WHOI,CP10CNSM
+Coastal Pioneer MAB Northern Surface Mooring,WHOI,CP11NOSM
+Coastal Pioneer MAB Southern Surface Mooring,WHOI,CP11SOSM
+Coastal Pioneer MAB Central Shallow Water Mooring,WHOI,CP12CNSW
+Coastal Pioneer MAB Western Shallow Water Mooring,WHOI,CP12WESW
+Coastal Pioneer MAB Eastern Profiler Mooring,WHOI,CP13EAPM
+Coastal Pioneer MAB Northern Profiler Mooring,WHOI,CP13NOPM
+Coastal Pioneer MAB Southern Profiler Mooring,WHOI,CP13SOPM
+Coastal Pioneer MAB Northeastern Profiler Mooring,WHOI,CP14NEPM
+Coastal Pioneer MAB Southeastern Profiler Mooring,WHOI,CP14SEPM
+Coastal Pioneer MAB Mobile Assets - Gliders,Teledyne Webb,CP15MOAS
+Coastal Pioneer MAB Mobile Assets - AUVs,Hydroid,CP16MOAS
```

Comment:
Looks good.

Reply:
The additions look good, but it looks like they should be above as this list looks like it is supposed to be alphabetical.  So I think the MAB stuff should go in front of NES.

Reply:
ARgggggghhh ok - rearranged; plus had to rearrange some NES ones that were not alphabetical...